### ISSUE TITLE
Apply checksums

### ISSUE THREAD

**Issue by fatz** [dcos/tf_dcos_core#10]
Currently there is no checksum matching against the downloaded DC/OS. This should be improved as an attacker might be able to inject code.

- Add checksums to the download urls
- Proof checksums / even better signatures ( to be evaluated with product management )
- Think about signatures / checksums between bootstrap and cluster nodes.

**Comment by bernadinm** [comment]
Yes, this is a trivial add to verify checksums. The risk is very minimal however because we are in control of path of which the `dcos_generate_config.sh` is currently being sourced from here: https://github.com/dcos/tf_dcos_core/blob/master/open/uri/download-variables.tf.
